py3status/modules/do_not_disturb.py

```diff
@@ -1,104 +1,225 @@
 # -*- coding: utf-8 -*-
 """
-Turn on and off dbus notifications.
+Toggle Do Not Disturb notifications.
 
-A left mouse click will toggle the state of this module.
+This module supports dunst, xfce4-notifyd, and lxqt-notificationd.
+For best results, don't install more than one notification server.
 
 Configuration parameters:
-    dunst_signal_off: Signal to send to the dunst process to disable "Do Not Disturb."
-        (default 'SIGTERM')
-    dunst_signal_on: Signal to send to the dunst process to enable "Do Not Disturb."
-        (default 'SIGUSR1')
-    format: Display format for the "Do Not Disturb" module.
-        (default '{state}')
-    notification_manager: The process name of your notification manager.
-        (default 'dunst')
-    refresh_interval: Refresh interval to use for killing notification manager process.
-        (default 0.25)
-    state_off: Message when the "Do Not Disturb" mode is disabled.
-        (default 'OFF')
-    state_on: Message when the "Do Not Disturb" mode is enabled.
-        (default 'ON')
+    cache_timeout: refresh interval for this module; for xfce4-notifyd
+        (default 10)
+    format: display format for this module
+        (default '{server} [\?if=state&color=good DND|\?color=bad DND]')
+    server: specify notification server to use, otherwise auto
+        (default None)
+    signals: specify 2-tuples of off/on signals; for dunst
+        (default ('SIGTERM', 'SIGUSR1'))
+    state: specify state to use on startup, otherwise 'last'
+        False: off, True: on, 'last': last, None: none
+        (default 'last')
+
+Format placeholders:
+    {server} notification server name, eg dunst, xfce4-notifyd, lxqt-notif...
+    {state} do not disturb state, eg False, True
 
 Color options:
-    color_bad: "Do Not Disturb" mode is enabled.
-    color_good: "Do Not Disturb" mode is disabled.
+    color_bad: Do Not Disturb is disabled
+    color_good: Do Not Disturb is enabled
+
+Examples:
+```
+# try few themes
+do_not_disturb {
+    format = '{server} [\?if=state&color=good  ON|\?color=bad OFF]'
+    # format = '{server} [\?if=state&color=good 1|\?color=bad 0]'
+    # format = '[\?if=state&color=good DO NOT DISTURB|\?color=lightgray DISTURB]'
+    # format = '{server} {state}'
```

Comment:
I don't like the example like this if you want four then add them as four examples with discriptions.

For me I should be able to copy paste the examples into my config and them work (excluding api_keys etc)

Reply:
Will change.